src/components/augmint-ui/icon/styles.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+const BaseIcon = `
+    height: 1em;
```

Comment:
I think we should replace all these `em` values with `rem` so we have better control over the actual evaluation.